### DISCUSSION TITLE
Inputs Documentation

### DISCUSSION THREAD

**Discussion by Kotlin-GDE** [tailwindlabs/tailwindcss#11571] [Ideas]
hi

provide a full documentation for forms (inputs components)

**Comment by mybearworld** [comment]
Are you talking about [this](https://github.com/tailwindlabs/tailwindcss-forms#readme)?

**Reply by Kotlin-GDE** [reply]
yes

**Reply by mybearworld** [reply]
Alright then... There's your docs
